exalearn-ui/src/app/components/assign-test-modal/assign-test-modal.component.ts

```diff
@@ -15,6 +16,8 @@ export class AssignTestModalComponent {
 	sendEmail = true;
 	minDate: Date;
 	maxDate: Date;
+	levels = EnglishLevels;
+	levelsValues = Object.values(this.levels);
```

Comment:
Try to get rid of this line. It is not needed.

Consider creating a custom pipe or a better method to read directly from ENUM   :)